receiver/prometheusreceiver/internal/transaction.go

```diff
@@ -159,6 +159,12 @@ func (tr *transaction) Commit() error {
 	if tr.useStartTimeMetric {
 		// AdjustStartTime - startTime has to be non-zero in this case.
 		if tr.metricBuilder.startTime == 0.0 {
+			// Unable to adjust start time because of missing start time metric
```

Comment:
There's three comments saying basically the same thing here (including the log message). Can probably remove/consolidate 1-2 of them

Reply:
Rephrased both comments. Does it look better now?